awspricing/__init__.py

```diff
@@ -28,69 +24,48 @@ def _fetch_offers():
     if offers is not None:
         return offers
 
-    resp = session.get(OFFER_BASE_URL + OFFER_INDEX_ENDPOINT)
-    resp.raise_for_status()
-    offers = resp.json()['offers']
+    for product in DEFAULT_AWS_PRODUCTS:
+        _fetch_offer(product)
```

Comment:
Did you consider whether we could fetch these lazily instead? e.g if someone only uses EC2, could we avoid ever fetching RDS?

Reply:
Removed this default product list